frontend/src/components/order/order.tsx

```diff
@@ -0,0 +1,86 @@
+import type { ClosedOrderType, OpenOrderType } from "../../lib/types";
+import { getFormattedCurrency } from "../../lib/util";
+
+import styles from "./order.module.css";
+
+export function ClosedOrder(props: ClosedOrderType) {
```

Comment:
We should go ahead and separate out all of the react elements into their own class

Reply:
Done